helm_deploy/hmpps-interventions-ui/templates/_envs.tpl

```diff
@@ -0,0 +1,62 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Environment variables for web and worker containers
+*/}}
+{{- define "deployment.envs" }}
+env:
+  - name: INGRESS_URL
+    value: "https://{{ .Values.ingress.host }}/"
+
+  - name: CLIENT_ID
+    valueFrom:
+      secretKeyRef:
+        name: {{ template "app.name" . }}
+        key: CLIENT_ID
+
+  - name: CLIENT_SECRET
+    valueFrom:
+      secretKeyRef:
+        name: {{ template "app.name" . }}
+        key: CLIENT_SECRET
+
+  - name: APPINSIGHTS_INSTRUMENTATIONKEY
+    valueFrom:
+      secretKeyRef:
+        name: {{ template "app.name" . }}
+        key: APPINSIGHTS_INSTRUMENTATIONKEY
+
+  - name: SESSION_SECRET
+    valueFrom:
+      secretKeyRef:
+        name: {{ template "app.name" . }}
+        key: SESSION_SECRET
+
+  - name: REDIS_HOST
+    valueFrom:
+      secretKeyRef:
+        name: hmpps_template_typescript_elasticache_redis
+        key: primary_endpoint_address
+
+  - name: REDIS_AUTH_TOKEN
+    valueFrom:
+      secretKeyRef:
+        name: hmpps_template_typescript_elasticache_redis
+        key: auth_token
+
+  - name: REDIS_TLS_ENABLED
+    value: {{ .Values.env.REDIS_TLS_ENABLED }}
+    value: "true"
+
+  - name: HMPPS_AUTH_URL
+    value: {{ .Values.env.HMPPS_AUTH_URL | quote }}
+
+  - name: TOKEN_VERIFICATION_API_URL
```

Comment:
In `feature.env`, this variable has the same host as `HMPPS_AUTH_URL` with a different path.

This intrigues me
- this name is functional -- we're using it for token verification
- the other (`HMPPS_AUTH_URL`) is a system name -- should it also be functional, like `AUTH_URL`?